R/ml_random_forest.R

```diff
@@ -41,11 +53,41 @@ ml_random_forest <- function(x,
     ml.options = ml.options
   )
 
+  col.sample.rate <- ensure_scalar_double(col.sample.rate, allow.null = TRUE)
```

Comment:
Should we allow users to specify strategies by name directly? (e.g. supply `"onethird"` explicitly)